modules/tidal.py

```diff
@@ -159,13 +161,15 @@ def enable_peripheral_I2C():
 def init_lcd():
     _LCD_PWR_ALWAYS.off() # this is mandatory even if LCD is disabled using lcd_power_off() - having this pin high significantly increases power consumption
     lcd_backlight_on()
-    lcd_power_on()
     _LCD_CS.off()
     display.init()
     # Set up scrolling parameters, if anyone wants to use them
     display.vscrdef(40, 240, 40)
     display.vscsad(40)
 
+init_lcd()
```

Comment:
I think these would be better to be explicit in boot.py? In fact, boot.py already calls init_lcd? I don't think we'd want to immediately turn it off either? boot.py generally wants to immediately do stuff with it. But the changes to lcd_power_on look great if they don't have any additional power cost (and it sounds like they don't)